addons/mail/static/src/components/follower_subtype_list/follower_subtype_list.xml

```diff
@@ -9,7 +9,7 @@
                         <h4 class="modal-title">
                             Edit Subscription of <t t-esc="followerSubtypeList.follower.name"/>
                         </h4>
-                        <i class="o_FollowerSubtypeList_closeButton close fa fa-times" aria-label="Close" t-on-click="_onClickCancel"/>
+                        <i class="o_FollowerSubtypeList_closeButton close fa fa-times" aria-label="Close" t-on-click="followerSubtypeList.follower.onClickCancel()"/>
```

Comment:
This is not the cancel button, this should be another handler for on click close. (commit:  Move onClickCancel)